src/Auth0.ManagementApi/Clients/EmailProviderClient.cs

```diff
@@ -38,7 +40,7 @@ public Task<EmailProvider> ConfigureAsync(EmailProviderConfigureRequest request)
         /// <returns>A <see cref="Task"/> that represents the asynchronous delete operation.</returns>
         public Task DeleteAsync()
         {
-            return Connection.DeleteAsync<object>("emails/provider", null, null);
+            return Connection.SendAsync<object>(HttpMethod.Delete, BuildUri("emails/provider"), null);
```

Comment:
Does the `null` need to be explicit here? Seeing this in a few places but only just registered.

Reply:
I thought about making `null` optional on Send but I didn't want anyone to accidently wipe out user data should the method be called by a badly written or generated stub. This way you have to explicitly say that the body should be null - just in case.